packages/vite/src/node/config.ts

```diff
@@ -1090,7 +1090,7 @@ export function loadEnv(
     const path = lookupFile(envDir, [file], true)
     if (path) {
       const parsed = dotenv.parse(fs.readFileSync(path), {
-        debug: !!process.env.DEBUG || undefined
+        debug: process.env.DEBUG?.includes('dotenv') || undefined
```

Comment:
We should use the same scheme we use for other switches here, no? So someone can debug only dotenv in vite and not in other libs
```suggestion
        debug: process.env.DEBUG?.includes('vite:dotenv') || undefined
```

Reply:
That's right 👌